LDAR_Sim/model_code/stationary_company.py

```diff
@@ -72,11 +72,11 @@ def __init__(self, state, parameters, config, timeseries, module_name):
             n_fixed = int(site['fixed_sensors'])
             for i in range(n_fixed):
                 self.crews.append(
-                    fixed_crew(
+                    stationary_crew(
                         state, parameters, config, timeseries, site, self.deployment_days,
                         id=site['facility_ID'] + '-' + str(i + 1)))
                 self.timeseries['fixed_cost'][self.state['t'].current_timestep] += \
-                    self.config['up_front_cost']
+                    self.config['cost']['upfront']
```

Comment:
1. Should upfront costs be accrued on a per crew basis?
2. Up front costs should be possible for other solutions as well that want to account for capital expenditures! E.g. each OGI crew up front cost could be $150,000 for cameras!